tools/run_tests/run_tests.py

```diff
@@ -371,6 +371,7 @@ def test_specs(self):
       tests_json = json.load(tests_json_file)
     environment = dict(_FORCE_ENVIRON_FOR_WRAPPERS)
     environment['PYVER'] = '2.7'
+    environment['PYTHONPATH'] = os.path.abspath('src/python/gens')
```

Comment:
When new tests are launched, they are launched with ```subprocess.Popen()``` with an explicit environment, which causes the child process to not inherit the current environment variables. 

Because test_specs() is more or less the entry point for a specific test, the only place we could have introduced a conflicting PYTHONPATH is with
```environment = dict(_FORCE_ENVIRON_FOR_WRAPPERS)```